### PR TITLE
CACTUS-768 :: Disabled Select Options

### PR DESCRIPTION
**Pull request by Dhalton**
Ticket & UAT: https://repayonline.atlassian.net/browse/CACTUS-768

I cleaned up some stuff the tests while I was in there, mostly just type signatures we don't need and unnecessary calls to `animationRender()`.

### CODE REVIEW

File: modules/cactus-web/src/Select/Select.test.tsx [hunk 1]
```diff
@@ -392,15 +421,100 @@ describe('component: Select', (): void => {
...
+      fireEvent.keyDown(listbox, { keyCode: KeyCodes.DOWN, charCode: KeyCodes.DOWN })
+      expect(listbox.getAttribute('aria-activedescendant')).toBe(thirdActive.id)
+
+      fireEvent.keyDown(listbox, { keyCode: KeyCodes.UP, charCode: KeyCodes.UP })
+      expect(listbox.getAttribute('aria-activedescendant')).toBe(secondActive.id)
+      fireEvent.keyDown(listbox, { keyCode: KeyCodes.UP, charCode: KeyCodes.UP })
+      expect(listbox.getAttribute('aria-activedescendant')).toBe(firstActive.id)
+    })
+
+    test('sets the active descendant to the first/last option on HOME/END keydown, respectfully', () => {
```
Comment (wilysword) [inline modules/cactus-web/src/Select/Select.test.tsx:465]: *respectively

File: modules/cactus-web/src/Select/Select.test.tsx [hunk 2]
```diff
@@ -392,15 +421,100 @@ describe('component: Select', (): void => {
...
+      const lastOption = getByText('last')
+
+      fireEvent.keyDown(listbox, { keyCode: KeyCodes.END, charCode: KeyCodes.END })
+      expect(listbox.getAttribute('aria-activedescendant')).toBe(lastOption.id)
+
+      fireEvent.keyDown(listbox, { keyCode: KeyCodes.HOME, charCode: KeyCodes.HOME })
+      expect(listbox.getAttribute('aria-activedescendant')).toBe(firstOption.id)
+    })
+
+    test('sets the active descendant to the first/last non-disabled option on HOME/END keydown, respectfully', () => {
```
Comment (wilysword) [inline modules/cactus-web/src/Select/Select.test.tsx:484]: *respectively

**Review by wilysword — APPROVED** [review]
👏

File: modules/cactus-web/src/Select/Select.test.tsx (hunk 1 above)
Comment (Dhalton) [reply modules/cactus-web/src/Select/Select.test.tsx:465]: 🤦  duh moment. I knew it sounded weird in my head when I was writing it, too...

**Review by daniloPenaR — APPROVED** [review]
👍

**Pull request merged**